jina/executors/indexers/vector.py

```diff
@@ -272,21 +311,24 @@ def query(self, keys: 'np.ndarray', top_k: int, *args, **kwargs) -> Tuple[Option
             Distance (the smaller the better) is returned, not the score.
 
         """
+        # better name. cannot change function signature for now
+        vector_queries = keys
```

Comment:
I am not convinced by the renaming ;) It verschlimmbessert (german word for "worse but better") things.

Reply:
Do you mean the approach, or the names themselves? I found the original ones didn't really match

Reply:
Both. But in this case also the variable name. I see where it comes from, but in the search context, they are `keys`.

Reply:
Ok, reverted

Reply:
The conversation in the related issue will eventually bring a solution for renaming.